tests/link/pytorch/test_basic.py

```diff
@@ -301,3 +302,20 @@ def test_pytorch_MakeVector():
     x_fg = FunctionGraph([], [x])
 
     compare_pytorch_and_py(x_fg, [])
+
+
+def test_pytorch_ifelse():
+    p1_vals = np.r_[1, 2, 3]
+    p2_vals = np.r_[-1, -2, -3]
+
+    a = scalar("a")
+    x = ifelse(a < 0.5, tuple(np.r_[p1_vals, p2_vals]), tuple(np.r_[p2_vals, p1_vals]))
+    x_fg = FunctionGraph([a], x)
+
+    compare_pytorch_and_py(x_fg, np.array([0.2], dtype=config.floatX))
+
+    a = scalar("a")
+    x = ifelse(a < 0.4, tuple(np.r_[p1_vals, p2_vals]), tuple(np.r_[p2_vals, p1_vals]))
+    x_fg = FunctionGraph([a], x)
```

Comment:
If you want to test twice, to cover the false case, do a case where the multiple outputs are not something that can be stacked (say `(pt.zeros((3, 5), pt.ones(2,))`.

I would have the other test return a single output, to test the single output case as well